### PR TITLE
Create Hrmn97.json

### PR DESCRIPTION
**Pull request by Hrmn97**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4028"></a>

### CODE REVIEW

File: data/Hrmn97.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Full Stack Web developer || MERN",
+  "avatar": "https://github.com/Hrmn97.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Harman9765",
+      "icon": "FaTwitter"
+    }
```
Comment (swagataroy30) [inline data/Hrmn97.json:12]: comma is missing in line 12

File: data/Hrmn97.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/h_97singh/",
+      "icon": "FaInstagram"
+    }
```
Comment (swagataroy30) [inline data/Hrmn97.json:17]: comma is missing in line 17

File: data/Hrmn97.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    {
+      "name": "COnnect with me on GitHub",
+      "url": "https://github.com/Hrmn97",
+      "icon": "FaGithub"
+    }
```
Comment (swagataroy30) [inline data/Hrmn97.json:22]: comma is missing in line 22

**Review by dan-mba — CHANGES_REQUESTED** [review]
To fix the JSON errors, add a `,` to the end of lines 12, 17, & 22

File: data/Hrmn97.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      "url": "https://twitter.com/Harman9765",
+      "icon": "FaTwitter"
+    }
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/h_97singh/",
+      "icon": "FaInstagram"
+    }
+    {
+      "name": "COnnect with me on GitHub",
```
Comment (swagataroy30) [inline data/Hrmn97.json:19]: fix type that should be 'Connect'

**Pull request closed without merging**